anvil/anvil.go

```diff
@@ -90,7 +92,12 @@ func (a *Anvil) Start(ctx context.Context) error {
 	anvilPortCh := make(chan uint64)
 
 	// Handle stdout/stderr
-	//  - TODO: Figure out best way to dump into logger. Some hex isn't showing appropriately
+	logFile, err := os.CreateTemp("", fmt.Sprintf("anvil-chain-%d-", a.cfg.ChainId))
```

Comment:
extra `-` at the end?

Reply:
ah wait no actually the extra `-` is there becase `CreateTemp` appends a random string of characters at the end for name collision